### PR TITLE
Chore/#437 adapt to changes of industry core

### PR DESCRIPTION
**Pull request by mkanal**
Add concept for industry core changes

### CODE REVIEW

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+   BOMLifecycleDetector -->> IRS : retrun BOMLifecyle
+end
+
+````
+
+#### Configuration of parameter 'digitalTwinType'
+Parameter is configurable for dDTR instance and has to be configured for integration test.
+
+https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/f438fe96a7ae1f1b920e8e4fb4114fb7af32643d/charts/registry/values.yaml#L51
+https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/f438fe96a7ae1f1b920e8e4fb4114fb7af32643d/charts/registry/README.md#:~:text=externalSubjectIdWildcardAllowedTypes
```
Comment (ds-mmaul) [inline docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:123]: shorten link

**Review by ds-mmaul — CHANGES_REQUESTED** [review]
I have left my comments. As this is a wip document, I would approve it as it is. Feel free to decide if and which changes you want to make and then let me know to approve it

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
+# IndustryCore Changes #437
+
+| Key           | Value                                                                                                                                                      |
+|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Autor         | @ds-mkanal                                                                                                                                                 |
+| Creation date | 26.01.2024                                                                                                                                                 |
+| Ticket Id     | [#437](https://github.com/eclipse-tractusx/traceability-foss/issues/437) [#357](https://github.com/eclipse-tractusx/item-relationship-service/issues/357) [#437]( |
```
Comment (mkanal) [inline docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:7]:
```suggestion
| Ticket Id     | [#437](https://github.com/eclipse-tractusx/traceability-foss/issues/437) [#357](https://github.com/eclipse-tractusx/item-relationship-service/issues/357)  |
```

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+4. [Definition of the Industry Core](#definition-of-the-industry-core)
+5. [Glossary](#glossary)
+6. [References](#references)
+
+# Overview
+
+The "Industry Core" is a fundamental element in the C-X network or system architecture.
+It provides essential functions for identification, traversal, data flow, configuration, and communication in a modular and
+interconnected environment.
+The goal is to make components identifiable and discoverable at type and instance levels.
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:23]: No numerical list neededn

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+|---|---------|-----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| AAS |         |                                                     |                                                                                                                                                   |
+| PartAsPlanned | 1.0.1   | Mandatory                                           | [PartAsPlanned 1.0.1](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.part_as_planned/1.0.1)                        |
+ |PartAsPlanned | 2.0.0   | Optional (Mandatory for next version of CX-0126)    | [PartAsPlanned 2.0.0](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.part_as_planned/2.0.0)                        |
+| SingleLevelBomAsPlanned| 1.1.0   | Mandatory                                           | [SingleLevelBomAsPlanned 1.1.0](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.single_level_bom_as_planned/1.1.0)] |
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:48]: Please provide a suggestion.

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+| SingleLevelBomAsPlanned| 2.0.0   | Optional (Mandatory for next version of CX-0126)    | [SingleLevelBomAsPlanned 2.0.0](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.single_level_bom_as_planned/2.2.0)] |
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:49]:
```suggestion
| SingleLevelBomAsPlanned| 2.0.0   | Optional (Mandatory for next version of CX-0126)    | [SingleLevelBomAsPlanned 2.0.0](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.single_level_bom_as_planned/2.2.0) |
```

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+|Parameter | Availability |
+|----|------------------|
+|jisNumber | Mandatory |
+|parentOrderNumber | Optional |
+|jisCallDate | Optional |
+|partInstanceId | Mandatory |
+
+# Definition of the Industry Core
+
+- [CX-0127-IndustryCorePartInstance#1.0.0](https://github.com/catenax-eV/product-standardization-prod/tree/CX-0127-IndustryCorePartInstance-v1.0.0/standards/CX-0127-IndustryCorePartInstance)
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:152]: You do not have permission to access

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 7]
```diff
@@ -0,0 +1,246 @@
...
+
+## Configure the necessary enablement services
+- EDC Setup (BPN-S/BPN-L, Policy Handling of the Core)
+
+## Sending of notifications
+- Header
+- EDC-Assets
+
+## Core Elements
+- [CX-0002-DigitalTwinsInCatenaX](https://github.com/catenax-eV/product-standardization-prod/tree/CX-0127-IndustryCorePartInstance-v1.0.0/standards/CX-0002-DigitalTwinsInCatenaX)
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:229]: You do not have permission to access

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 8]
```diff
@@ -0,0 +1,246 @@
...
+| BPN | Business Partner Number | A BPN is the unique identifier of a partner within Catena-x. |
+|  |Part Instance|A part instance is a physically produced instance (e.g. serialized part, batch, just-in-sequence-part) of a part type. |
+|  |Part Type| A part type is a generic (not physically produced) part on material- or catalog-level as a representation for a designed part.|
+| VAN | Vehicle Anonymised Number | A number mapped 1:1 to VIN, but pseudonomised.|
+
+# References
+- [CX-0126 Industry Core Part Type Standard 1.0.0](https://github.com/catenax-eV/product-standardization-prod/blob/CX-0126-IndustryCorePartType-v1.0.0/standards/CX-0126-IndustryCorePartType/1.0.0/CX-0126-Industry-Core-Part-Type.pdf)
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:244]: You do not have permission to access

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 9]
```diff
@@ -0,0 +1,246 @@
...
+- [CX-0126 Industry Core Part Instance Standard 1.0.0](https://github.com/catenax-eV/product-standardization-prod/blob/CX-0127-IndustryCorePartInstance-v1.0.0/standards/CX-0127-IndustryCorePartInstance/1.0.0/CX%20-%200127%20Industry%20Core%20Part%20Instance%201.0.0.pdf)
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:245]: You do not have permission to access

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md [hunk 10]
```diff
@@ -0,0 +1,246 @@
...
+   IRS -->> BOMLifecycleDetector : throw Exception (BOMLifecylce could not be detected)
+   BOMLifecycleDetector -->> IRS : retrun BOMLifecyle
+end
+
+````
+
+#### Configuration of parameter 'digitalTwinType'
+Parameter is configurable for dDTR instance and has to be configured for integration test.
+
+https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/f438fe96a7ae1f1b920e8e4fb4114fb7af32643d/charts/registry/values.yaml#L51
```
[the remark this answers is not in the document]
Comment (mkanal) [reply docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:122]: Why?

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md (hunk 1 above)
Comment (mkanal) [inline docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:123]:
```suggestion
- [externalSubjectIdWildcardAllowedTypes](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/f438fe96a7ae1f1b920e8e4fb4114fb7af32643d/charts/registry/README.md#:~:text=externalSubjectIdWildcardAllowedTypes)
```

File: docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md (hunk 10 above)
Comment (mkanal) [inline docs/concept/#437-adapt-to-changes-of-industry-core/#437-adapt-to-changes-industry-core.md:122]:
```suggestion
[values.yaml](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/f438fe96a7ae1f1b920e8e4fb4114fb7af32643d/charts/registry/values.yaml#L51)
```

**Review by ds-mmaul — APPROVED** [review]
LGTM, Thanks

**Pull request merged**